samples/dect/dect_phy/hello_dect/Kconfig

```diff
@@ -11,6 +11,15 @@ config CARRIER
 	   The availability of the channels and the exact regulation to use them varies in different countries.
 	   See ETSI TS 103 636-2 5.4.2 for the calculation.
 
+config BAND_GROUP_INDEX
```

Comment:
I'm fine also with this but please have a comment in CONFIG_CARRIER that BAND_GROUP_INDEX should be tuned as well if using band 4.  Add comment to Readme?

Alternatively, I was thinking that if this could this be set automatically based on set CONFIG_CARRIER? Basically only band #4 is currently supported on low bands.

Reply:
Added range for carrier when band group index 1 is set.